spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/services/backendService/SCExchangeBackend.java

```diff
@@ -0,0 +1,414 @@
+/**
+ * Copyright 2017 Boundless, http://boundlessgeo.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and limitations under the License
+ */
+package com.boundlessgeo.spatialconnect.services.backendService;
+
+
+import android.util.Log;
+
+import com.boundlessgeo.spatialconnect.SpatialConnect;
+import com.boundlessgeo.spatialconnect.config.SCConfig;
+import com.boundlessgeo.spatialconnect.config.SCFormConfig;
+import com.boundlessgeo.spatialconnect.config.SCFormField;
+import com.boundlessgeo.spatialconnect.config.SCRemoteConfig;
+import com.boundlessgeo.spatialconnect.geometries.SCGeometry;
+import com.boundlessgeo.spatialconnect.geometries.SCSpatialFeature;
+import com.boundlessgeo.spatialconnect.mqtt.SCNotification;
+import com.boundlessgeo.spatialconnect.scutilities.HttpHandler;
+import com.boundlessgeo.spatialconnect.scutilities.Json.SCObjectMapper;
+import com.boundlessgeo.spatialconnect.services.SCConfigService;
+import com.boundlessgeo.spatialconnect.services.SCDataService;
+import com.boundlessgeo.spatialconnect.services.SCSensorService;
+import com.boundlessgeo.spatialconnect.services.SCService;
+import com.boundlessgeo.spatialconnect.services.authService.SCAuthService;
+import com.boundlessgeo.spatialconnect.stores.ISyncableStore;
+import com.boundlessgeo.spatialconnect.stores.SCDataStore;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.vividsolutions.jts.geom.Geometry;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import rx.Observable;
+import rx.functions.Action1;
+import rx.functions.Func1;
+import rx.functions.Func2;
+
+/**
+ * SCExchangeBackendService to enable Boundless Exchange to serve as a backend to the
+ * spatialconnect-android-sdk.
+ */
+public class SCExchangeBackend implements ISCBackend {
+
+    private static final String LOG_TAG = SCExchangeBackend.class.getSimpleName();
+    private static final String SERVICE_NAME = "SC_EXCHANGE_BACKEND_SERVICE";
```

Comment:
We probably can remove the service name, this was the old approach instead of using a backendType, its technically not a service in the sense of the other services.

Reply:
Good catch...I'll remove it